packages/forma-36-react-components/src/components/Dropdown/Dropdown/Dropdown.tsx

```diff
@@ -197,6 +199,7 @@ export class Dropdown extends Component<DropdownProps, DropdownState> {
         {toggleElement}
         {this.state.isOpen && (
           <DropdownContainer
+            className={dropdownContainerClassName}
```

Comment:
Do we also need to pass className to the DropdownContainer on [L176](https://github.com/contentful/forma-36/pull/234/files#diff-33e7e56e26c1eb1c22349a4e4ab6ce9fR176)?